frontend/sw360-portlet/src/main/webapp/html/components/detailRelease.jsp

```diff
@@ -58,29 +59,61 @@
-        && (window.location = createDetailURLfromReleaseId (this.options[this.selectedIndex].value) );">
-            <core_rt:forEach var="releaseItr" items="${component.releases}">
-                <option <core_rt:if test="${releaseItr.id == releaseId}"> selected </core_rt:if>
-                        value="${releaseItr.id}"><sw360:ReleaseName release="${releaseItr}" />
-                </option>
-            </core_rt:forEach>
-        </select>
-        <span class="pull-right">
-        <input type="button" id="edit" data-release-id="${releaseId}" value="Edit" class="addButton">
-        <sw360:DisplaySubscribeButton email="<%=themeDisplay.getUser().getEmailAddress()%>" object="${release}" id="SubscribeButton" />
-    </span>
-    </label>
-    </p>
+      <div id="header">
+        <p class="pageHeader"> <span class="pageHeaderBigSpan"> Component: ${component.name}</span>
+
+            <span class="dropdown">
+                <core_rt:forEach var="releaseItr" items="${component.releases}">
+                    <core_rt:if test="${releaseItr.id == releaseId}"> <button id="dropbtn" class="dropbtn"> ${releaseItr.name} ${releaseItr.version} <span class="arrow-down"/>  </button> </core_rt:if>
+                </core_rt:forEach>
+                <span id="releaseDropdown" class="dropdown-content">
+                <core_rt:forEach var="releaseItr" items="${component.releases}">
+                    <a href="#" onclick="window.location=createDetailURLfromReleaseId( '${releaseItr.id}')">${releaseItr.name} ${releaseItr.version}
+                        <core_rt:if test="${releaseItr.clearingState.value == 4}">  <span class="sw360CircleOK"> </span> </core_rt:if>
+                        <core_rt:if test="${releaseItr.clearingState.value == 0}"> <span class="sw360CircleAlert"> </span> </core_rt:if>
+                        <core_rt:if test="${releaseItr.clearingState.value == 1}">  <span class="sw360CircleAlert"> </span> </core_rt:if>
+                        <core_rt:if test="${releaseItr.clearingState.value == 2}">  <span class="sw360CircleWarning"> </span> </core_rt:if>
+                        <core_rt:if test="${releaseItr.clearingState.value == 3}">  <span class="sw360CircleWarning"> </span> </core_rt:if>
+                    </a>
+                </core_rt:forEach>
+                </span>
+            </span>
+            <span class="pull-right">
+                <input type="button" id="edit" data-release-id="${releaseId}" value="Edit" class="addButton">
+                <sw360:DisplaySubscribeButton email="<%=themeDisplay.getUser().getEmailAddress()%>" object="${release}" id="SubscribeButton" />
+            </span>
+        </p>
+        </div>
+
     <core_rt:set var="inReleaseDetailsContext" value="true" scope="request"/>
     <%@include file="/html/components/includes/releases/detailOverview.jspf"%>
 </core_rt:if>
 <script>
+    /* variables used in releaseTools.js ... */
+    var releaseIdInURL = '<%=PortalConstants.RELEASE_ID%>',
```

Comment:
these variables are already set in the edit.jsp of the component

Reply:
I tested it and all the variables are necessary.

Reply:
Yes you are right, these variables on the edit page of a component are not set for this page.
I think there's still room for improvement how we generate the sw360 liferay links.